lib/commands/general.js

```diff
@@ -39,6 +39,16 @@ commands.getDeviceTime = async function () {
   }
 };
 
+commands.setDeviceTime = async function (dateTime) {
+  log.info('Attempting to set device date and time');
```

Comment:
I assume a better place for such stuff would be appium-adb module. WDYT @imurchie ?

Reply:
I don't mind making that change, these are my first PRs.  I was basically working alongside getDeviceTime

Reply:
Yes, this ought to be done in the `appium-adb` module. We should not really ever be calling `adb.shell` outside of there, if possible (I know we break that all over the place, but shouldn't :( ).